### PR TITLE
Feat : added functionality

### PR DESCRIPTION
**Pull request by Omkar-Chaskar**
Added Add new note , Home Notes, Note archive , note trash and note restore functionality

### CODE REVIEW

File: src/component/home.js [hunk 1]
```diff
@@ -5,57 +5,91 @@ import {
   MdLabelOutline,
   MdOutlineArchive,
   MdDeleteOutline,
+  MdPushPin,
+  MdDelete
 } from "react-icons/md";
+import { useNotes } from "../context/noteProvider";
```
Comment (TheAmlanSaikia) [inline src/component/home.js:5]: You can use index  for shorter route path.

**Review by TheAmlanSaikia — APPROVED** [review]
Awesome Code. good To merge. Left one suggestion.
Things I liked are-
Use of barrel export.
Clean Code.
Use of good variable names.
Gooed spacing and identation.

File: src/component/home.js (hunk 1 above)
Comment (Omkar-Chaskar) [reply src/component/home.js:5]: Thanks . I will do it in refactor

**Review by rahulmistry751 — APPROVED** [review]
Good clean code

**Pull request merged**